__tests__/index.tsx

```diff
@@ -170,7 +170,7 @@ describe('serialize', () => {
   })
 
   test('strips imports & exports', async () => {
-    const result = await renderStatic(`import foo from 'bar'
+    const result = await renderStatic(`import foobar from 'bar'
```

Comment:
To be able to remove the imports it now needs to be able to successfully parse the JS, but it can't since there's 2x `foo` constant assignments - therefore renamed the import.